sources/packages/backend/libs/sims-db/src/entities/student-loan-balances.model.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  RelationId,
+} from "typeorm";
+import { ColumnNames, TableNames } from "../constant";
+import { RecordDataModel } from "./record.model";
+import { Student } from "./student.model";
+import { numericTransformer } from "../transformers/numeric.transformer";
+
+/**
+ * Stores files information and content from NSLSC
+ * Loan Balance file.
+ */
+@Entity({ name: TableNames.StudentLoanBalances })
+export class StudentLoanBalances extends RecordDataModel {
+  @PrimaryGeneratedColumn()
+  id: number;
+  /**
+   * Student id associated with this Loan Balance.
+   */
+  @RelationId(
```

Comment:
Please avoid adding the `RelationId` since we will remove them all.

Reply:
I noticed that the decorator was removed but the entire property is not needed. Please remove it.